lightly_studio/tests/api/routes/api/test_video.py

```diff
@@ -23,12 +23,13 @@ def test_get_all_videos(test_client: TestClient, db_session: Session) -> None:
         ],
     )
 
-    response = test_client.get(
+    response = test_client.post(
```

Comment:
Should we write more tests for this now? I'm not sure, since it's an initial version and I would prefer to be more certain about it.